### PR TITLE
add download page design

### PR DESCRIPTION
**Pull request by brandonhaslegs**

### CODE REVIEW

**Review by juliendonck — COMMENTED** [review]
- should this not just link to the download page? It feel strange that I go to another section where I gotta find and click it again.
Screenshot 2020-11-06 at 15 15 59
- Does the download page not need binaries for each operating system? Linux/Windows/Mac?
- Wonder if this page needs a title like the other ones, feels a bit sad. Maybe it can just be super functional. 
- the app screenshot kinda feels random to me here.
- I had imagined something like the protocol section for the download page -> 3 identical boxes side by side for web and stacked for mobile with a big header above.

**Comment by rudolfs** [comment]
> Does the download page not need binaries for each operating system? Linux/Windows/Mac?

We only support Linux and Mac at the moment.

**Comment by juliendonck** [comment]
> > Does the download page not need binaries for each operating system? Linux/Windows/Mac?
> 
> We only support Linux and Mac at the moment.

And each of these will have their individual buildkite link, right?

**Comment by rudolfs** [comment]
> And each of these will have their individual buildkite link, right?

Well, I wonder if we can link directly to the images or not?

If we just link to the buildkite release then it would be only one link, and the user has to dig to get to the images.
Both Linux and macOS are on the same page but in different sections:

Screenshot 2020-11-06 at 15 29 30

I wonder if the links that we can copy from the buildkite page are stable (i.e. will not stop working after some time) and actually work if we link to them directly from our website?

Linux:
https://buildkite.com/organizations/monadic/pipelines/radicle-upstream/builds/5170/jobs/19496657-a935-4468-b595-6330b8f1c7de/artifacts/767e4ee3-3dfb-4571-96ae-c0e79a2fe8c9

macOS:
https://buildkite.com/organizations/monadic/pipelines/radicle-upstream/builds/5170/jobs/4d402caf-fc9e-4af0-b6a9-3c255e1b42a5/artifacts/9095e734-d42a-4aff-a948-384c335c6657

Maybe @kim  can chime in?

File: docs/index.html [hunk 1]
```diff
@@ -238,7 +238,7 @@ <h3>Join the  Community</h3>
               <h3>Try out the beta</h3>
               <p>Interested in integrating Radicle into your code collaboration experience?</p>
               <br>
-              <a target="_blank" href="/"><span>Download the beta</span> ↓</a>
+              <a target="_blank" href="/downloads.html"><span>Download the beta</span> ↓</a>
```
Comment (juliendonck) [inline docs/index.html:241]:
```suggestion
              <a href="/downloads.html"><span>Download the beta</span> ↓</a>
```

File: pages/index.html.mustache [hunk 2]
```diff
@@ -144,7 +144,7 @@
               <h3>Try out the beta</h3>
               <p>Interested in integrating Radicle into your code collaboration experience?</p>
               <br>
-              <a target="_blank" href="/"><span>Download the beta</span> ↓</a>
+              <a target="_blank" href="/downloads.html"><span>Download the beta</span> ↓</a>
```
Comment (juliendonck) [inline pages/index.html.mustache:147]:
```suggestion
              <a href="/downloads.html"><span>Download the beta</span> ↓</a>
```

**Review by juliendonck — APPROVED** [review]
Looks great

**Pull request merged**